yarn.lock

```diff
@@ -59,15 +59,15 @@
 
 "@aws-sdk/abort-controller@3.53.0":
   version "3.53.0"
-  resolved "https://registry.yarnpkg.com/@aws-sdk/abort-controller/-/abort-controller-3.53.0.tgz#9e114f54bf52220bec279e5fd5f83a8ea76437b0"
+  resolved "https://registry.npmjs.org/@aws-sdk/abort-controller/-/abort-controller-3.53.0.tgz"
```

Comment:
How come all these changed?

Reply:
oh thats super weird let me check on that

Reply:
hmm ok all the changes i did made this worse lol... i removed `yarn.lock` and did a fresh `make bs` and the diffs became larger 😅

Reply:
at least it seems like it changed the other way now (npm -> yarn, which seems like that should be the case since we're using yarn?)